src/makeRobot.js

```diff
@@ -39,6 +39,55 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward(value = 1) {
+      if (value > 0) {
```

Comment:
don't use abstract variable names like 'value, result, number' etc.
better rename for more understandable code.
```suggestion
    goForward(move = 1) {
      if (move > 0) {
```